example-app/src/main/java/com/adyen/checkout/example/ui/card/CardViewState.kt

```diff
@@ -0,0 +1,14 @@
+package com.adyen.checkout.example.ui.card
+
+import com.adyen.checkout.components.model.paymentmethods.PaymentMethod
+
+internal sealed class CardViewState {
+
+    object Loading : CardViewState()
+
+    data class Data(val paymentMethod: PaymentMethod) : CardViewState()
```

Comment:
Maybe something more expressive like `ShowComponent` would be better?

Reply:
Good suggestion